stories/module-analytics-components.stories.js

```diff
@@ -29,26 +29,6 @@ import { STORE_NAME } from '../assets/js/modules/analytics/datastore/constants';
 import {
 	accountsPropertiesProfiles,
 	goals,
-	dashboardAllTrafficArgs,
-	dashboardAllTrafficData,
-	pageDashboardAllTrafficArgs,
-	pageDashboardAllTrafficData,
-	dashboardPopularPagesArgs,
-	dashboardPopularPagesData,
-	pageDashboardBounceRateWidgetArgs,
-	pageDashboardBounceRateWidgetData,
-	pageDashboardUniqueVisitorsSparkArgs,
-	pageDashboardUniqueVisitorsSparkData,
-	pageDashboardUniqueVisitorsVisitorArgs,
-	pageDashboardUniqueVisitorsVisitorData,
-	dashboardBounceRateWidgetArgs,
-	dashboardBounceRateWidgetData,
-	dashboardGoalsWidgetArgs,
-	dashboardGoalsWidgetData,
-	dashboardUniqueVisitorsSparkArgs,
-	dashboardUniqueVisitorsVisitorArgs,
-	dashboardUniqueVisitorsSparkData,
-	dashboardUniqueVisitorsVisitorData,
```

Comment:
The data these stories use shouldn't need to change. If for some reason the data does need to be updated (curious why this would be), it should be updated in the fixture rather than inlined.

Reply:
Hey @aaemnnosttv, I've inlined the widget data based on [this comment from Felix](https://github.com/google/site-kit-wp/issues/2459#issuecomment-736892298) and to keep it consistent with the other widget tickets I've worked on.

Reply:
@aaemnnosttv The problem here is that fixtures aren't a great approach because the data is dependent on another value passed here, the `referenceDate`. I think that would justify its own issue though. For now, although I agree it's terrible it's most appropriate to inline the data since it's dependent and it's not a static "fixture".